tutorials/nbconverted/general_usage.py

```diff
@@ -0,0 +1,227 @@
+plate_map = (metadata_dir / "platemap_NF1_CP.csv").resolve(strict=True)
+
+# setting output paths
+sc_profile_path = out_dir / "nf1_single_cell_profile.csv.gz"
+anno_profile_path = out_dir / "nf1_annotated_profile.csv.gz"
+norm_profile_path = out_dir / "nf1_noramlzied_profile.csv.gz"
+feat_profile_path = out_dir / "nf1_features_profile.csv.gz"
+
+
+# ## Generating Merged Single-cell Morphology Dataset
+# 
+# In this section of the walkthrough, our goal is to load the NF1 dataset and create a merged single-cell morphology dataset.
+# 
+# Currently, the NF1 dataset is stored in an `sqlite` format, where each table represents a different compartment, such as Image, Cell, Nucleus, and Cytoplasm.
+# To achieve this, we will utilize the SingleCells class, which offers a range of functionalities specifically designed for single-cell analysis. You can find detailed documentation on these functionalities [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.cyto_utils.html#pycytominer.cyto_utils.cells.SingleCells).
+# 
+# However, for our purpose in this walkthrough, we will focus on using the SingleCells class to merge all the tables within the NF1 sqlite file into a merged single-cell morphology dataset.
+
+# ### Updating defaults
+# Before we proceed further, it is important to update the default parameters in the `SingleCells`class to accommodate the table name changes in our NF1 dataset.
+# 
+# Since the table names in our NF1 dataset differ from the default table names recognized by the `SingleCells` class, we need to make adjustments to ensure proper recognition of these table name changes.
+
+# In[19]:
+
+
+# update compartment names and strata
+strata = ["Image_Metadata_Well", "Image_Metadata_Plate"]
+compartments = ["Per_Cells", "Per_Cytoplasm", "Per_Nuclei"]
+
+# Updating linking columns for merging all compartments
+linking_cols = {
+    "Per_Cytoplasm": {
+        "Per_Cells": "Cytoplasm_Parent_Cells",
+        "Per_Nuclei": "Cytoplasm_Parent_Nuclei",
+    },
+    "Per_Cells": {"Per_Cytoplasm": "Cells_Number_Object_Number"},
+    "Per_Nuclei": {"Per_Cytoplasm": "Nuclei_Number_Object_Number"},
+}
+
+
+# Now that we have stored the updated the parameters, we can use them as inputs for SingleCells class to proceed with the merging of all the NF1 sqlite tables into a single consolidated dataset.
+
+# In[20]:
+
+
+# setting up sqlite address
+sqlite_address = f"sqlite:///{str(plate_data)}"
+
+# loading single cell morphology data into pycyotminer's SingleCells Object
+single_cell_profile = SingleCells(
+    sql_file=sqlite_address,
+    compartments=compartments,
+    compartment_linking_cols=linking_cols,
+    image_table_name="Per_Image",
+    strata=strata,
+    merge_cols=["ImageNumber"],
+    image_cols="ImageNumber",
+    load_image_data=True,
+)
+
+# mering all sqlite table into a single tabular dataset (csv)
+sc_profile = single_cell_profile.merge_single_cells(
+    sc_output_files=sc_profile_path, compression_options="gzip"
```

Comment:
```suggestion
    sc_output_file=sc_profile_path, compression_options="gzip"
```
I do believe the correct spelling of the parameter is `file` not `files`.